app/views/purchases/new.html.erb

```diff
@@ -0,0 +1,45 @@
+<section class="flex-column height-full">
```

Comment:
- [ ] When I try to save a transaction, it gives an error as shown below: 
![image](https://user-images.githubusercontent.com/58504655/236483365-e689cb58-458e-4f4c-984f-28448bf9f231.png)

Reply:
Hii thanks for the suggestion but as I have created a new transaction It is working fine.